lms/templates/courseware/dates.html

```diff
@@ -26,10 +26,19 @@
                 <h2 class="hd hd-2 date-title">
                     ${_("Important Dates")}
                 </h2>
+                % if not display_reset_dates_text:
+                    <div class="dates-banner">
+                        <div class="dates-banner-text">
+                            <strong>${_("We've built a suggested schedule to help you stay on track.")}</strong>
+                            ${_("But don't worry—it's flexible so you can learn at your own pace.")}
+                            ${_("If you happen to fall behind on our suggested dates, you'll be able to adjust them to keep yourself on track.")}
```

Comment:
i think this can be on the same line as the sentence before it.

Reply:
also, by default, `display_reset_dates_text` is going to evaluate to false, so this will show up even when the pls flag is turned off.  However, the dates tab as a whole doesn't appear with the pls flag off, so i think that should be good enough.  just thinking out loud